terraform/cloud.gov.tf

```diff
@@ -569,14 +569,38 @@ resource "aws_route53_record" "cdn_broker_delegate" {
   ]
 }
 
+# Start SitesUSA subomains
+/*
+    We are trying to keep this file as clean as possible so we are grouping
+    the subdomains here.  We should probably get our own file or something,
+    we have about 46 domains.
+ */
```

Comment:
Just to underline from our Slack convo, the values of the names may be arbitrary, but their organization is absolutely meaningful, and should be respected going forward. Cloud.gov DNS entries here, other entries in their own file, even if it's 100% CNAMEs to cloud.gov stuff.

Thanks!